src/package/package.ts

```diff
@@ -51,7 +51,7 @@ export const Package2Fields = [
   'IsOrgDependent',
   'ConvertedFromPackageId',
   'PackageErrorUsername',
-];
+]; // 'AppAnalyticsEnabled' is only available starting in api version 59.0. See getPackage2Fields below.
```

Comment:
@stephen-carter-at-sf I would suggest adding the field to the list now and then exclude it in function `this.getPackage2Fields` if API version < 59.